game/templates/game/board.html

```
@@ -454,6 +454,11 @@
             <div style="display: flex; flex-direction: column; gap: 12px;">
                 <button type="button" class="btn btn-gold" id="shareResultBtn" style="padding: 12px; font-size: 1rem;">📤 Share Result</button>
                 <button class="btn btn-gold" id="gameOverStartBtn" style="padding: 12px; font-size: 1rem;">Start New Game</button>
+                <button id="goHomeBtn" type="button" class="secondary-btn">
+                    ← Back to Home
+                </button>
+                    ← Back to Home
+                </button>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </button> ]

(tag-pair)